test/JDBC/expected/sys-configurations.out

```diff
@@ -2,13 +2,34 @@ SELECT * FROM sys.configurations;
 GO
```

Comment:
since sys.configuations uses UNINON internally, we really need to add order by clause here in the query to avoid flakiness

Reply:
Applicable at all the places where we are testing sys.configuations